proto/tendermint/types/types.proto

```diff
@@ -213,6 +213,7 @@ message BlockMeta {
   int64   block_size = 2;
   Header  header     = 3 [(gogoproto.nullable) = false];
   int64   num_txs    = 4;
+  DataAvailabilityHeader da_header = 5 [(gogoproto.nullable) = false];
```

Comment:
@liamsi, what was the thinking behind not using `[(gogoproto.nullable) = false]` for the DAH in the `tmproto.Block`? I used it here, but can revert.

Reply:
No particular reason. I want to use these gogoprote annotations as little as possible. Some of them break with the actual protobuf specification. 

@marbar3778 is there a good rule of thumb when to use and when to use them?

Reply:
Revisiting John's favorite topic, but the [deterministic protobuf spec](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-027-deterministic-protobuf-serialization.md) doesn't include any provisions for gogoproto annotations. In fact, if we stick to deterministic protobuf then `[(gogoproto.nullable) = false]` would be invalid.

Reply:
@adlerjohn, why do you think it is unmaintained?

Reply:
switched to not using gogo in https://github.com/lazyledger/lazyledger-core/pull/372/commits/3e3fa306441421fe78f96400f4737dcf899fdb93